lib/synapse/haproxy.rb

```diff
@@ -1030,19 +1030,33 @@ def generate_backend_stanza(watcher, config)
       ]
     end
 
+    def haproxy_exec(command)
+      info = nil
```

Comment:
i think you don't need to initialize this here; this seems like it was done just to have something for the `ensure` block to reference